### ISSUE TITLE
[ActionMenu] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1269]
### Feedback

❌ aria-controls should reference the menu element. Here it is referencing the <sp-popover> element which seems to actually be empty until it opens. Once the button opens aria-controls is no longer valid as the popup is moved out of the same shadow document.
？ Suggest removing aria-controls as it is not required per the spec and aria authoring practices for a menu button
？ Suggest modifying the aria-label on the Alternate Icon example in Doc so the text is used as the label of the button.

### Path forward

1. `aria-controls` usage is being updated in #1266 
2. same
3. Accessible label should be "Actions under the great". Remove `label` usage to achieve and move to recommended use of `<sp-icon-*>` usage:
   ```
       <sp-action-menu>
           <sp-icon-gear></sp-icon-gear>
           <span slot="label">Actions under the gear</span>
   ```

cc @jnurthen
